website/blog/2019-01-21-jest-24-new-website-babel-7-and-typescript-support.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Jest 24: New Website, simpler TypeScript setup, Improved Assertions and TODOs
+author: Simen Bekkhus
+authorURL: https://github.com/SimenB
+authorFBID: 100003004880942
+---
+
+Today we are happy to announce the next major release of Jest - version 24! It's been 4 months since the last minor release, and 8 months since Jest 23, so this upgrade is a big one, with something for everyone! Highlights include built-in support for TypeScript by upgrading the Jest internals to Babel 7, fixing some long standing issues with missing console output and performance issues when computing large diffs, and a brand new sparkling website. ✨
+
+For a full list of all changes see the [changelog](https://github.com/facebook/jest/blob/master/CHANGELOG.md).
+
+<!--truncate-->
+
+## New Website
+
+[@orta](https://twitter.com/orta) has provided a beautiful redesign of Jest's website, which has been implemented by community members [@montogeek](https://twitter.com/montogeek) and [@brainkim](https://github.com/brainkim).
+
+The aim of the redesign was to highlight more of what makes Jest awesome, and to decouple the idea that Jest is primarily a tool for testing React apps - you can use Jest with all sorts of projects and we want to make that obvious. You can read more about the ideas behind the redesign in [this issue](https://github.com/facebook/jest/issues/7265).
+
+## TypeScript support
+
+We've upgraded internally to Babel 7 for Jest 24, which comes with support for TypeScript projects. That means Jest can support transpiling TypeScript out of the box, as long as you configure Babel to use it with `@babel/preset-typescript`. One caveat to the default TypeScript support, similar to Flow, is that Babel will only strip out the type annotations to make your code valid JavaScript. It will _not_ typecheck your code.
```

Comment:
```suggestion
We've upgraded to Babel 7 internally for Jest 24, which comes with support for TypeScript projects. That means Jest can support transpiling TypeScript out of the box, as long as you configure Babel to use it with `@babel/preset-typescript`. One caveat to the default TypeScript support, similar to Flow, is that Babel will only strip out the type annotations to make your code valid JavaScript. It will _not_ typecheck your code.
```